content/js2/blocks/access/index.md

```diff
@@ -42,6 +42,19 @@ const profileData = {
 console.log(profileData.firstName); // logs "Francesco"
 ```
 
+Objects also allow looking up property values using square brackets, similar to arrays. Instead of an index, we use a string of the key of the property inside the square brackets:
```

Comment:
```suggestion
Objects also allow looking up property values using square brackets, similar to arrays. Instead of an index, we use a string of the key inside the square brackets:
```